### DISCUSSION TITLE
login not possible

### DISCUSSION THREAD

**Discussion by Gelbwichtel** [nicholas0g/InvoxiaWebClient#2] [Q&A]
Hi, i think i need your help.
After start npm i can connect to the login site.
But login is not possible. These logs are promted at the console. on the web site, there are no positive or negative infos desplayed.
![bild1](https://github.com/nicholas0g/InvoxiaWebClient/assets/71406061/4206e6e1-e5c7-44e8-8686-153096519f2b)

**Comment by Aquariu** [comment]
Hi there !

I've got the same error on my instance.
I originally installed this last year and I believe it was working great. I reinstalled the server recently and pulled everything anew.

I tried both the Docker version and the standalone "npm start" version. Both turn up the web interface correctly, (very easy setup) both lead to the same logs as @Gelbwichtel here. (either on the Docker logs or on the command line, depending on how you launch it).
I also needed to change the port just to check that I could do that. It works, but I still don't get past login.

Kind regards,
